README.md

```diff
@@ -130,13 +131,42 @@ loadConfig[MyClass](conf)
 ```
 
 
+## Integrating with other libraries
+
+### Internal Modules
+
+The core of PureConfig eschews unnessary dependencies. Separate modules exist to support types which are not part of the standard Scala and Java libraries.
+
+- [`pureconfig-enum`](modules/enum/) provides converters for enums generated by [julienrf's enum library](https://github.com/julienrf/enum).
```

Comment:
Minor (OCD-level, in fact) nitpick: in other places in the README we end each list item with a `;` except the last one, which ends in `.`. Can you do that here too?